core/api_views.py

```
@@ -0,0 +1,391 @@
+        end_date = request.GET.get('end_date')
+        format_type = request.GET.get('format', 'CSV')
+        
+        if not start_date or not end_date:
+            return Response({
+                'error': 'start_date and end_date required'
+            }, status=status.HTTP_400_BAD_REQUEST)
+        
+        result = audit_service.export_audit_trail(start_date, end_date, format_type)
+        return Response({'export_path': result}, status=status.HTTP_200_OK)
+        
+    except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)